lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -831,10 +831,25 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
         if certificate_info[0] == 'Y':
             TASK_LOG.info(
                 u'Student is marked eligible_for_certificate'
-                u'(user=%s, course_id=%s, grade_percent=%s gradecutoffs=%s, allow_certificate=%s, is_whitelisted=%s)',
+                u'(user=%s, course_id=%s, grade_percent=%s letter_grade=%s gradecutoffs=%s, allow_certificate=%s, '
+                u'is_whitelisted=%s)',
                 student,
                 course_id,
                 course_grade.percent,
+                course_grade.letter_grade,
+                course.grade_cutoffs,
+                student.profile.allow_certificate,
+                student.id in whitelisted_user_ids
+            )
+        else:
+            TASK_LOG.info(
+                u'Student is marked ineligible_for_certificate'
+                u'(user=%s, course_id=%s, grade_percent=%s letter_grade=%s gradecutoffs=%s, allow_certificate=%s, '
+                u'is_whitelisted=%s)',
+                student,
+                course_id,
+                course_grade.percent,
+                course_grade.letter_grade,
                 course.grade_cutoffs,
                 student.profile.allow_certificate,
                 student.id in whitelisted_user_ids
```

Comment:
You can easily DRY this by having just one log statement and passing `certificate_info[0]` as a parameter.